cmd/cluster-agent/subcommands/flare/command.go

```diff
@@ -75,6 +75,10 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 }
 
 func run(log log.Component, config config.Component, cliParams *cliParams) error {
+	warnings := config.Warnings()
+	if warnings != nil && warnings.Err != nil {
+		fmt.Fprintln(color.Output, color.YellowString("%v", warnings.Err))
```

Comment:
can we send to `stderr` since it is a warning/error?  
Also, can we add some context to the log line, maybe something like

```suggestion
		fmt.Fprintln(color.Output, color.YellowString("Config parsing warning: %v", warnings.Err))
```

Reply:
Code updated.